SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -994,7 +994,8 @@ void USpatialNetDriver::ServerReplicateActors_ProcessPrioritizedActors(UNetConne
 				// or it's an editor placed actor and the client hasn't initialized the level it's in
 				if (Channel == nullptr && GuidCache->SupportsObject(Actor->GetClass()) && GuidCache->SupportsObject(Actor->IsNetStartupActor() ? Actor : Actor->GetArchetype()))
 				{
-					if (Actor->GetClass()->HasAnySpatialClassFlags(SPATIALCLASS_NotSpatialType))
+					if (Actor->GetClass()->HasAnySpatialClassFlags(SPATIALCLASS_NotSpatialType) ||
+						!Actor->GetClass()->HasAnySpatialClassFlags(SPATIALCLASS_ExplicitSpatialType))
```

Comment:
I believe these flags are mutually exclusive. So we should just be able to check for the existence of ExplicitSpatialType.

Reply:
Good Spot. Done